README.md

```diff
@@ -21,7 +21,8 @@ If you are new to the world of ACAPs take a moment to check out
   - [APIs](#apis)
 - [Installation and usage](#installation-and-usage)
   - [Installation](#installation)
-    - [Native ACAP application](#native-acap-application)
+  - [Native ACAP application for versions >1.2.0](#native-acap-application)
```

Comment:
The TOC looks a bit strange - did you update it manually? There is an extension for VS Code that can do it automatically